applications/SwimmingDEMApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -396,7 +396,7 @@ void  AddCustomUtilitiesToPython(pybind11::module& m){
 
 
     py::class_<BassetForceTools> (m, "BassetForceTools")
-        .def(py::init<>())
+        .def(py::init<Parameters&>())
```

Comment:
`Kratos::Parameters` should be passed by-value according to riccardo due to how internally the storage of the json is handled

Reply:
This will be addressed in its own PR